extensions/2.0/Khronos/KHR_displaymapping_pq/README.md

```diff
@@ -0,0 +1,400 @@
+0.0691 0.9195 0.0114  
+0.0164 0.0880 0.8956  
+
+A HDR capable display is defined as having at least 10 bits per pixel for each colorchannel. 
+
+### To SDR capable display
+
+If the framebuffer format or colorspace is not known, or none is available that preserves range, precision and color gamut then lower range framebuffer with sRGB colorspace may be used.  
+This is to allow for compatibility with displays that does not support higher range and/or compatible colorspaces.  
+It also allows for implementations where the details of the framebuffer is not known or available.  
+
+A SDR capable display is defined as having less than 10 bits per pixel for each colorchannel.  
+
+For SDR output,  a range extension value of 46.42 and gamma of 2.4 shall be used.  
+
+
+### OOTF
+
+Opto-optical-transfer-function  
+Input signal to the OOTF is the scene linear light. The OOTF maps relative scene linear light to display linear light.  
+For more information see the ITU BT.2100 specification.  
+
+Resulting linear scene light values (the pixel output) shall be display mapped according to the parameter `Reference PQ OOTF` of ITU BT.2100   
+The opto-optical reference transform shall be applied, the reference transform compatible with both SDR and HDR displays is described in ITU BT.2390 with the use of range extension and gamma values.  
+
+E′ = G709[E] = pow(1.099 (rangeExtension * E), 0.45) – 0.099 for 1 > E > 0.0003024
+               267.84 * E for 0.0003024 ≥ E ≥ 0
+FD = G1886[E'] = pow(100 E′, gamma)
+
+E is the linear scene light value. 
+
+Where the rangeExtension and gamma values shall be set according to HDR or SDR display.  
+
+```
+HDR
+range extension = 59.5208  
+gamma = 2.4  
+```
+
+```
+SDR
+range extension = 46.42  
+gamma = 2.4  
+```
+
+
+**Implementation Notes**
+
+Pseudocode for BT.2100 reference OOTF  
+
+```
+BT_2100_OOTF(color, rangeExponent, gamma) {  
+    if (color <= 0.0003024) {  
+        nonlinear = 267.84 * color;  
+    else {  
+        nonlinear = 1.099 * pow(rangeExponent * color, 0.45) - 0.099;  
+    }  
+    return 100 * pow(nonlinear, gamma);
+}  
+```
+
+Where this is calculated per RGB component, note that the color value in this equation must be in range 0.0 - 1.0
+It may be estimated that the display will not have the ability to output dark levels in the region of 0.0003024 in which case implementations may ignore that condition and use the same operator for the whole range 0.0 - 1.0  
+
+The incoming color value shall first be adjusted by aperture as detailed in [See Scene aperture](#scene-aperture) 
+
+
+### OETF
+
+Opto-electrical-transfer-function  
+Input signal to the PQ OETF is the scene linear light. The PQ OETF maps scene linear light to non-linear PQ signal value.  
+For more information see the ITU BT.2100 specification.  
+
+After the OOTF is applied the OETF shall be applied, this will yield a non linear output-signal in the range [0.0 - 1.0] that is stored in the display buffer.  
```

Comment:
I'm not sure why you want the OOTF step? The OOTF computes values in display light, and the OETF, if fed display light values, should simply yield scene light values again, but we already have scene light values in the glTF case.

Reply:
The OOTF is OETF->EOTF, I think in your case it "happens" to work as some kind of tone mapper because the OOTF is not symmetrical, but that's not its purpose. You are effectively adding a tone mapper without calling it that.

Reply:
It is the reference OOTF consistent with ITU broadcast practises and is taken from ITU BT 2390.

> I think in your case it "happens" to work as some kind of tone mapper because the OOTF is not symmetrical, but that's not its purpose.

Why do you think that?

> You are effectively adding a tone mapper without calling it that.

I am adding a reference OOTF (from ITU BT 2390) , which is meant to provide the reference output - anything added to this would be considered artistic intent (tone mapper)